source/quickstart.rst

```diff
@@ -6,22 +6,24 @@ Introduction
 
 Welcome to Manim DSL's documentation!
 
-Manim DSL is a Domain Specific Language that is made to help lecturers make animations for teaching data structures
-and algorithms
+Manim DSL is a domain specific language that helps educators create animations for teaching students about data structures
+and algorithms.
 
 Here is an example:
 
-.. code:: none
+.. code:: javascript
 
     let stack: Stack = new Stack;
     stack.push(5);
     stack.pop();
 
-Which would produce the video:
+This would produce the video:
 
+.. raw:: html
 
-<-- INSERT VIDEO -->
-
+    <div style="position: relative; padding-bottom: 56.25%; height: 0; overflow: hidden; max-width: 100%; height: auto;">
+        <iframe src="../media/intro.mp4" frameborder="0" allowfullscreen style="position: absolute; top: 0; left: 0; width: 80%; height: 80%;"></iframe>
```

Comment:
Does this need to be an iframe? Is it possible to use video instead?

Reply:
An HTML video? I don't mind, but what's the advantage?

Reply:
I've looked this up a bit, I think we can use this: https://github.com/ManimDSL/manimdsl.github.io/blob/master/source/conf.py#L54 property in the conf file to serve static files, and then it won't be relative paths